### ISSUE TITLE
Push task can't converged

### ISSUE THREAD

**Issue by Lhy-code** [qgallouedec/panda-gym#103]
I ran the train_push.py(DDPG+HER) and visualize the loss and success rate results, it turns out that the two network did not converged and keep vibrating, and success rate also keep 0%.
I tried smaller learning rate but still not work, wonder how you can successfully train a good model? Thanks

![Image](https://github.com/user-attachments/assets/e7c8fe2b-6128-4067-85df-7866e0a79ea9)
